sdcm/utils/common.py

```diff
@@ -242,14 +244,17 @@ def download_file(self, link, dst_dir):
 
 
 def get_latest_gemini_version():
+    """search latest version of gemini tool"""
```

Comment:
https://www.python.org/dev/peps/pep-0257/

**tl;dr:** capitalize first letter and add period to the end.

But from my pov docstring is redundant here: `get_latest_gemini_version` is self-explanatory name enough.

Reply:
Removed